api/envoy/extensions/matching/common_inputs/network/v3/network_inputs.proto

```diff
@@ -95,3 +96,19 @@ message TransportProtocolInput {
 // [#extension: envoy.matching.inputs.application_protocol]
 message ApplicationProtocolInput {
 }
+
+// Input that matches the URIs in the SAN field of the peer certificate for a downstream.
+// [#extension: envoy.matching.inputs.uri_san]
```

Comment:
API question: SAN field is a list of names. Many matchers in Envoy pick the first item in the list. Is the the intent here to "check for any name in the list", and if so, wouldn't we need https://github.com/envoyproxy/envoy/issues/20666 as well?

Reply:
Currently, I will populate SAN field in a comma-delimited list, and I will document it in API.